### ISSUE TITLE
Add missing databases releases in our changelog

### ISSUE THREAD

**Issue by ipfaze** [Scalingo/documentation#2196]
Thanks to @brandon-welsch, we found out that there are databases releases that are not mentioned in our changelog.

For example, the PostgreSQL last changelog release was made on 11 August 2022 for version `12.11.0-3` as you can see below:
![image](https://github.com/Scalingo/documentation/assets/34447253/16ff1b4c-b0b6-403c-878b-aac7b3aa3f3b)
Since then, multiple PostgreSQL versions were released.

:warning: PostgreSQL may not be the only database that missed some releases.

An investigation must be done to determine which releases are missing and add them afterwards in our changelog section.

**Comment by yohann-bacha** [comment]
Okay, but why would we do such a refactoring of versions which are not available to use anymore? (just to know the reason)

**Comment by EtienneM** [comment]
I don't think this is really important. It would be a really boring job to make the diff between all databases versions available on Scalingo and the changelog entries. Moreover I don't really see the advantages of doing this work. Hence I think we can close this issue. What do you think @ipfaze?

**Comment by ipfaze** [comment]
We discussed about it during our team retrospective and we still want to tackle it.

**Comment by brandon-welsch** [comment]
@ipfaze what's the state of this?
